keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -370,7 +373,8 @@ class ApplicationService(
       build = artifactInstance.buildMetadata
         ?: artifactSupplier.parseDefaultBuildMetadata(artifactInstance, artifact.versioningStrategy),
       git = artifactInstance.gitMetadata
-        ?: artifactSupplier.parseDefaultGitMetadata(artifactInstance, artifact.versioningStrategy)
+        ?: artifactSupplier.parseDefaultGitMetadata(artifactInstance, artifact.versioningStrategy),
+      lifecycleEvents = lifecycleEventService.getEventsForArtifactAndVersion(artifact, artifactInstance.version)
```

Comment:
I'm curious to see how those events will be shown in the UI. @erikmunson , do you have any thoughts about it?